### PR TITLE
docs: developer tutorial

### LINKED ISSUE #221: Documentation for ownership-provider

### PR DESCRIPTION
**Pull request by homura**
# What Changed

Resolve #221

Preview at https://github.com/ckb-js/nexus/blob/docs-tutorial/docs/tutorial.md

A tutorial for developer, to show how to create a transaction via `FullOwnershipProvider` to communicate with Nexus

This PR dependent on 

- https://github.com/ckb-js/nexus/pull/227
- https://github.com/ckb-js/nexus/pull/255

## Motivation

A step-by-step tutorial can effectively lower the barrier to entry for developers and give them a sense of accomplishment

## Change Type

Indicate the type of change your pull request is:

- [x] `documentation`
- [ ] `patch`
- [ ] `minor`
- [ ] `major`

### CODE REVIEW

**Review by zhangyouxin — APPROVED** [review]

File: docs/tutorial.md [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+![img.png](tutorial-assets/tx-injected.png)
+
+### Step 6 - Pay For The Transaction Fee
+
+We'll pay for the transaction fee by using the cell we just injected, so we need to set `autoInject` to `false` and
+specify the output index of the cell we want to pay for the fee.
+
+```js
+// Pay for the transaction fee
+txSkeleton = await provider.payFee(txSkeleton, { autoInject: false, byOutputIndexes: [1] });
```
Comment (IronLu233) [inline docs/tutorial.md:133]: I think we should tell developer the outputs[2] is change output before set `byOutputIndexes = [1]`

File: docs/tutorial.md [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+```
+
+![tx-paid](tutorial-assets/tx-paid.png)
+
+### Step 7 - Sign The Transaction
+
+If all goes well, Nexus will ask you to sign the transaction.
```
Comment (IronLu233) [inline docs/tutorial.md:140]: Confirm the transaction detail and submit your password for unlocking your wallet.

File: docs/tutorial.md (hunk 1 above)
Comment (homura) [reply docs/tutorial.md]: commented for the `byOutputIndexes`

File: docs/tutorial.md (hunk 2 above)
Comment (homura) [reply docs/tutorial.md]: Added

**Review by IronLu233 — APPROVED** [review]

**Pull request merged**